text/1946-intra-rustdoc-links.md

```diff
@@ -330,14 +358,9 @@ For disambiguation markers using an `@`, in implied shortcut links
 you can use a space instead of the `@`. In other words, `[struct Foo]`
 is fine (and preferred).
```

Comment:
It doesn't look like this was ever implemented so should be removed from the RFC to avoid confusion.